### DISCUSSION TITLE
Grupos para atividade - aula 02

### DISCUSSION THREAD

**Discussion by igordsm** [Insper/open-dev#770] [Announcements]
Enunciado em https://insper.github.io/open-dev/lessons/02-software-and-communities/

**Comment by cemmanuelsr** [comment]
NeoVIM (Caio e Cícero)

**Comment by rafikli** [comment]
Kdenlive (Abel e Rafael)

**Comment by JorasOliveira** [comment]
Blender (Jorás e António)

**Comment by nataliaqmc** [comment]
VsCode (Natália e Jõao Andrade)

**Comment by igordsm** [comment]
Ordem dos grupos definida para segunda. Fechando a discussão
